bin/cron/upload_to_mapbox.rb

```diff
@@ -32,14 +34,16 @@ def put_file_on_s3(tileset_path, credentials)
   obj.upload_file(tileset_path)
 end
 
+# Return upload response code so that it can be logged
```

Comment:
Is this returning the response code or just a boolean?

Reply:
Good catch. At one point it was returning the response code but I changed it.